doc/content/verification_and_validation/val-2f.md

```diff
@@ -0,0 +1,100 @@
+# val-2f
+
+# Modelling self-damaged tungsten effects on deuterium transport
+
+## Test Description
+
+The case under study involves the use of recrystallized polycrystalline tungsten (PCW) samples, which are subjected to ion irradiation and subsequent analysis using thermal desorption spectroscopy (TDS). The primary objective is to determine how damage influences deuterium trapping and release. This case is drawn from [!cite](dark2024modelling).
+
+The TDS process is simulated using TMAP8 in a 1D tungsten sample with a thickness of 0.8 mm. The TDS simulation consisted of three phases: implantation, resting, and desorption.
+
+During the charging phase, deuterium is continuously implanted into the tungsten sample over a period of 72 hours. The temperature is maintained at 370 K throughout this phase. The surface is exposed to a constant flux of $\phi=5.79\times 10^{19}$ atoms/m$^2$/s, corresponding to a total fluence of $1.5\times 10^{25}$ atoms/m$^2$. The implantation profile follows a Gaussian distribution centered at the mean implantation depth of $R_p=0.7$ nm, with a standard deviation of $\sigma = 0.5$ nm :
+
+\begin{equation}
+    S(x) = \frac{1}{\sigma \sqrt{2\pi}} \exp\left( -\frac{(x - R_p)^2}{2\sigma^2} \right) \cdot \phi_{\text{surface}}(t)
+\end{equation}
+
+where the surface flux function is given by:
+
+\begin{equation}
+    \phi_{\text{surface}}(t) =
+    \begin{cases}
+        \phi, & t < 72 h \\
+        0, & \text{otherwise}
+    \end{cases}
+\end{equation}
+
+The implantation distribution is illustrated in [val-2f_implantation_distribution]. It highlights the need for mesh refinement on the order of $\sigma$ in the implantation region to accurately resolve the profile and capture the sharp gradient of the source term.
```

Comment:
What does `n the order of $\sigma$` mean? Please clarify. Maybe also explain why you plotted Rp+5sigma on the figure. Otherwise it is there without context.